SQLCreate.yml

```diff
@@ -1,13 +1,11 @@
 # Description
 # ===========
-# This playbook create a SQL server and an instance of SQL Database,
-# azure_rm_sqlserver, azure_rm_sqldatabase module is not available in v2.4, but in azure_preview_modules role,
-# so you need to run "ansible-galaxy install Azure.azure_preview_modules" to install the role to get lastest Ansible modules.
+# This playbook create a SQL server and an instance of SQL Database.
+# Note: you have to set variable sqlserver_name to a globally unique name, otherwise SQL server creation may fail
+#
 
 ---
 - hosts: localhost
-  roles:
-    - Azure.azure_preview_modules
   vars:
```

Comment:
do you want to randomize sqlserver name so that  ci  could pass?

Reply:
yes, i was thinking about that. do you think we should have random names in samples? or makes samples too complex?